tests/test_interface_props.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
```

Comment:
If you keep this as a separate file

1. Please generalize it beyond properties while scoping it to session.  For example, you could name it `test_session_intf.py` .
2. Pull in `test_intf_container` from `test_frames.py`